symplyphysics/laws/thermodynamics/entropy_derivative_via_volume_derivative.py

```diff
@@ -0,0 +1,105 @@
+from sympy import (
+    Eq,
+    Derivative,
+    symbols as sym_symbols,
+    Function as SymFunction,
+    solve,
+)
+from symplyphysics import (
+    symbols,
+    units,
+    Quantity,
+    Symbol,
+    Function,
+    validate_input,
+    validate_output,
+)
+from symplyphysics.core.expr_comparisons import expr_equals
+from symplyphysics.laws.thermodynamics import gibbs_energy_differential
+
+# Description
+## Maxwell relations are a set of equations that unite the most common thermodynamic quantities between
+## one another. They are derived from the fundamental themodynamic relations featuring differentials
+## of thermodynamic potentials, and this method of derivation is called the method of thermodynamic
+## potentials.
+
+# Law: (dS/dp)_T = -(dV/dT)_p
+## S - entropy
+## p - pressure
+## V - volume
+## T - absolute temperature
+## (d/dp)_T - partial derivative w.r.t. pressure at constant temperature
+## (d/dT)_p - partial derivative w.r.t. temperature at constant pressure
+
+# Conditions
+## - Changes in particle count are not taken into account and it is assumed to be constant.
+
+entropy = Function("entropy", units.energy / units.temperature)
+pressure = Symbol("pressure", units.pressure)
+volume = Function("volume", units.volume)
+temperature = symbols.thermodynamics.temperature
+
+law = Eq(
+    Derivative(entropy(temperature, pressure), pressure),
+    -1 * Derivative(volume(temperature, pressure), temperature)
+)
+
+# Derive from expression of Gibbs energy differential
+
+_gibbs_energy_expr = sym_symbols("gibbs_energy", cls=SymFunction)(temperature, pressure)
+
+# Since Gibbs energy is a differentiable function of all its parameters, the can apply
+# the symmetry of second derivatives, which is that for any sufficiently differentiable
+# function one can exchange the order of taking partial derivatives: d(df/dx)/dy = d(df/dy)/dx.
+# The expression for the second derivative of Gibbs energy must be the same whether one first
+# takes it w.r.t. temperature and then w.r.t. pressure or the other way round.
+
+_gibbs_energy_diff_temperature = gibbs_energy_differential.law.rhs.subs({
+    gibbs_energy_differential.temperature_change: 1,
+    gibbs_energy_differential.pressure_change: 0,
```

Comment:
Thanks for the description above. But is it safe to do so? You set `pressure_change` to 0 and in the code below you differentiate over pressure.

Reply:
It's safe. The differential $`dp`$ applies only to the calculation of the first-order differential $`dG`$. It doesn't imply that the pressure stays constant in the whole expression. The factor at $`dp`$, which is basically $`\left(\frac{dG}{dp}\right)_T`$, can itself depend on pressure.